.github/workflows/ci.yml

```
@@ -74,11 +74,13 @@ jobs:
           npm install --force @biomejs/cli-linux-x64 @rollup/rollup-linux-x64-gnu
       
       - name: Run tests
-        run: npm run test:run
+        run: npm run test:ci
+        env:
+          NODE_OPTIONS: "--max-old-space-size=6144"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)